src/components/SignUp.js

```diff
@@ -1,74 +1,164 @@
-import React from 'react';
+import React, {useState} from 'react';
 import { Link } from "react-router-dom";
 import picLogScreen from '../assets/picLogScreen.png'
-import { Container, Form, Button, Row, Col} from 'react-bootstrap';
+import { Container, Form, Button} from 'react-bootstrap';
 import './SignUp.css';
+import { axiosInstance } from '../util/config';
+import * as Yup from 'yup';
+import { useNavigate } from "react-router-dom";
+
 
 export default function SignUp() {
+    const [firstName, setFirstName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [gender, setGender] = useState('');
+    const [email, setEmail] = useState('');
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+    const [repeatPassword, setRepeatPassword] = useState('');
+    const [error, setError] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
```

Comment:
These 9 `useState`s could have been reduced to only 1

Reply:
I just like it this way for me.